crates/cranelift/Cargo.toml

```diff
@@ -29,6 +29,7 @@ object = { workspace = true, features = ['write'] }
 thiserror = { workspace = true }
 cfg-if = { workspace = true }
 wasmtime-versioned-export-macros = { workspace = true }
+regex = "1.10"
```

Comment:
Is this dependency required for the string literal formatting?

Reply:
yes